errtracker/errtracker.go

```diff
@@ -119,6 +135,7 @@ func Report(snap, errMsg, dupSig string, extra map[string]string) (string, error
 		"CoreSnapdBuildID":   coreBuildID,
 		"Date":               timeNow().Format(time.ANSIC),
 		"Snap":               snap,
+		"PartialDpkgUpdate":  detectPartialDpkgUpdate(),
```

Comment:
StaleSnapdApparmorConffile maybe? Also we should not add it if its empty I think.